app/static/app/js/components/EditTaskPanel.jsx

```diff
@@ -38,35 +38,51 @@ class EditTaskPanel extends React.Component {
       }
 
       this.nodesRequest = 
-        $.getJSON("/api/processingnodes/?online=True", json => {
+        $.getJSON("/api/processingnodes/?has_available_options=True", json => {
```

Comment:
Consider using `fetch` (and its corresponding polyfill) to eliminate one of the reasons you might be depending on jQuery.